salt/kube-apiserver/apiserver.jinja

```diff
@@ -17,7 +17,11 @@ KUBE_API_PORT="--insecure-port=8080 --secure-port={{ pillar['api']['int_ssl_port
 KUBE_ETCD_SERVERS="--etcd-cafile={{ pillar['ssl']['ca_file'] }} \
                    --etcd-certfile={{ pillar['ssl']['kube_apiserver_crt'] }} \
                    --etcd-keyfile={{ pillar['ssl']['kube_apiserver_key'] }} \
+{% if not salt['grains.get']('node_removal_in_progress', False) %}
                    --etcd-servers={{ salt.caasp_etcd.get_endpoints() }}"
+{% else %}
+                   --etcd-servers={{ salt.caasp_etcd.get_endpoints(skip_removed=True) }}"
+{% endif %}
```

Comment:
This `if` should not be necessary: it wold be enough with the `skip_removed=True`, as [this parameter will skip all the nodes that are being removed](https://github.com/kubic-project/salt/blob/queue_etc_hosts_updates/salt/_modules/caasp_etcd.py#L160-L161).

Reply:
updated